datajunction-server/datajunction_server/api/cubes.py

```diff
@@ -139,9 +141,10 @@ async def get_cube_dimension_values(  # pylint: disable=too-many-locals
         submitted_query=translated_sql.sql,
         async_=async_,
     )
+    request_headers.update({"Cache-Control": cache_control})
```

Comment:
The cache control headers should already be in `request_headers` if they were passed in. So I wonder if this can be simplified by removing `cache_control: Annotated[str, Header()] = ""` from this method and just extracting the cache control values from `request.headers`, if it's even needed here? If it's not needed here and the sole purpose is just to pass over the headers to the query service, then we can just pass `request.headers` over instead of specifically pulling out cache control values.

Reply:
Great catch! I completely missed that but yes, from what I can tell we don't do anything with the cache control headers in the main service so these are just to pass along.